### DISCUSSION TITLE
how to use commitizen with a pre-commit when pre-commit-config.yaml is not in root folder

### DISCUSSION THREAD

**Discussion by thomasfinchbr** [commitizen-tools/commitizen#655] [Q&A]
Hi,

I am using pre-commit installed with the following option:

`pre-commit install --config=config\.pre-commit-config.yaml`

Except commitizen all tools I use I can add the option:

`- --config=config/pyproject.toml`

How could I configure commitezen to find the correct pre-commit config file? 

Thanks

**Comment by Lee-W** [comment]
I don't think we support it as of now. Feel free to send us a PR or open an issue if you're interested in it :)
